content_api_doc.md

```diff
@@ -0,0 +1,42 @@
+Content API
+===========
+
+**Understanding CNX URLs**
+
+The CNX URL structure is
+
+cnx.org/[book id][@version#]:[page id][@version #]/title
+
+ * All version numbers are optional. The lack of a version number will always return the latest version.
+ * In the browser, a short id is displayed for books and pages. There is also a longer UUID for books and pages. Either id will work in the API. The full UUID for a Book and the current Page can be found in the More Information tab at the bottom of any content.
+ * The title should not be included in the content API. request.
+
+**Book metadata and Table of Contents as json**
+
+  >GET http://archive.cnx.org/contents/[book ID].json
+
+Examples (long UUID and short id with version number)
+
+  >http://archive.cnx.org/contents/031da8d3-b525-429c-80cf-6c8ed997733a.json
```

Comment:
Doesn't need to be in this PR but what do people think about turning this into a doctest? ;)